crates/telio-wg/src/uapi.rs

```diff
@@ -34,6 +34,8 @@ pub struct Peer {
     pub public_key: PublicKey,
     /// Peer's endpoint with `IP address` and `UDP port` number
     pub endpoint: Option<SocketAddr>,
+    /// Mesh's IP addresses of peer
+    pub ip_addresses: Vec<IpAddr>,
```

Comment:
is PartaileEq, Eq implemntation corrent? if we will not receive ip_address from uapi_request, comparison can never equal, making this operation mute ?